src/sentry/static/sentry/app/utils/getStacktraceBody.jsx

```diff
@@ -0,0 +1,34 @@
+import rawStacktraceContent from 'app/components/events/interfaces/rawStacktraceContent';
+
+export default function getException(event) {
```

Comment:
We should name this function `getStacktraceBody` to match the filename.

Reply:
Ah yes, should be consistent. @billyvg do you have any reservations about naming it `getStacktraceBody`? I'm open to better options incase this feels like it doesn't apply well to the issue diff

Reply:
👍